utilities/write_batch_with_index/write_batch_with_index_internal.h

```diff
@@ -452,11 +544,25 @@ class WriteBatchWithIndexInternal {
   //   and return kMergeInProgress
   // If batch does not contain this key, return kNotFound
   // Else, return kError on error with error Status stored in *s.
-  static WriteBatchWithIndexInternal::Result GetFromBatch(
-      const ImmutableDBOptions& ioptions, WriteBatchWithIndex* batch,
-      ColumnFamilyHandle* column_family, const Slice& key,
-      MergeContext* merge_context, WriteBatchEntryComparator* cmp,
-      std::string* value, bool overwrite_key, Status* s);
+  Result GetFromBatch(WriteBatchWithIndex* batch, const Slice& key,
+                      std::string* value, bool overwrite_key, Status* s) {
+    return GetFromBatch(batch, key, &merge_context_, value, overwrite_key, s);
+  }
+  Result GetFromBatch(WriteBatchWithIndex* batch, const Slice& key,
+                      MergeContext* merge_context, std::string* value,
+                      bool overwrite_key, Status* s);
+  Status MergeKey(const Slice& key, const Slice* value, std::string* result,
+                  Slice* result_operand = nullptr) {
+    return MergeKey(key, value, merge_context_, result, result_operand);
+  }
+  Status MergeKey(const Slice& key, const Slice* value, MergeContext& context,
+                  std::string* result, Slice* result_operand = nullptr);
+
+ private:
+  DB* db_;  // Not owned
```

Comment:
I don't think any of these pointers are owned. Why only mark this one as unowned and not the others?